### DISCUSSION TITLE
[Feature] Add a "System Theme" that automatically selects light/dark mode based on system colorscheme

### DISCUSSION THREAD

**Discussion by benjamin051000** [immich-app/immich#8758] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Add an option that automatically selects light or dark mode based on the system theme.

This is a very common feature seen across many websites and apps. I have no idea how it is implemented: I presume there is an API you can call to query the system theme and adjust accordingly? Or some kind of event hook or something that triggers a callback when the system theme is changed. I can't imagine it would be too difficult to research. 

### Platform

- [ ] Server
- [X] Web
- [X] Mobile

**Comment by alextran1502** [comment]
It is already available on both platforms under user and app settings

**Reply by benjamin051000** [reply]
Ah, I see. And it removes the sun/moon icon. That works for me. Thank you!
